src/components/shared/NetworkSettings.css

```diff
@@ -0,0 +1,33 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+ .networkSettings {
```

Comment:
I think there is an extra indent here. You should be able to run `yarn lint-fix-css` to fix it.